lib/ProductOpener/Display.pm

```diff
@@ -10563,9 +10586,25 @@ sub display_product_history ($code, $product_ref) {
 			return display_tag_link('editors', $uid);
 		},
 		this_product_url => product_url($product_ref),
-		revisions => \@revisions
+		revisions => \@revisions,
+		product => $product_ref,
 	};
 
+	# Javascript needed to activate the product revert buttons in the edit history
+	if (has_permission($request_ref, "product_revert")) {
+		my $revert_confirm = lang("product_js_product_revert_confirm");
+		$scripts .= <<SCRIPTS
+<script>var revert_confirm_message = "$revert_confirm";</script>
+<script src="$static_subdomain/js/dist/product-history.js"></script>
+SCRIPTS
+			;
+
+		$initjs .= <<JS
+activate_product_revert_buttons_in_history();
+JS
+			;
```

Comment:
Don't we want to avoid this kind of JS inclusion (I find it quite annoying) ?

Why isn't it included in the template. Or we could maybe just use a specific class to trigger the JS activity.

Reply:
OK, I'm moving it.